include/OpenSpaceToolkit/Physics/Environment/Atmospheric/Earth/CSSISpaceWeather.hpp

```diff
@@ -0,0 +1,141 @@
+        Integer Ap3;    /// Planetary Equivalent Amplitude (Ap) for 0600-0900 UT.
+        Integer Ap4;    /// Planetary Equivalent Amplitude (Ap) for 0900-1200 UT.
+        Integer Ap5;    /// Planetary Equivalent Amplitude (Ap) for 1200-1500 UT.
+        Integer Ap6;    /// Planetary Equivalent Amplitude (Ap) for 1500-1800 UT.
+        Integer Ap7;    /// Planetary Equivalent Amplitude (Ap) for 1800-2100 UT.
+        Integer Ap8;    /// Planetary Equivalent Amplitude (Ap) for 2100-0000 UT.
+        Integer ApAvg;  /// Arithmetic average of the 8 Ap indices for the day.
+        Real Cp;        /// Cp or Planetary Daily Character Figure. A qualitative estimate of overall level of magnetic
+                        /// activity for the day determined from the sum of the 8 Ap indices. Cp ranges, in steps of
+                        /// one-tenth, from 0 (quiet) to 2.5 (highly disturbed).
+        Integer C9;     /// C9. A conversion of the 0-to-2.5 range of the Cp index to one digit between 0 and 9.
+        Integer ISN;    /// International Sunspot Number. Records contain the Zurich number through 1980 Dec 31 and the
+                        /// International Brussels number thereafter.
+        Real F107Obs;   /// Observed 10.7-cm Solar Radio Flux (F10.7). Measured at Ottawa at 1700 UT daily from 1947 Feb
+                        /// 14 until 1991 May 31 and measured at Penticton at 2000 UT from 1991 Jun 01 on. Expressed in
+                        /// units of 10-22 W/m2/Hz.
+        Real F107Adj;   /// 10.7-cm Solar Radio Flux (F10.7) adjusted to 1 AU.
+        String F107DataType;   /// Flux Qualifier.
+                               /// OBS: Observed flux measurement
+                               /// INT: CelesTrak linear interpolation of missing data
+                               /// PRD: 45-Day predicted flux
+                               /// PRM: Monthly predicted flux
+        Real F107ObsCenter81;  /// Centered 81-day arithmetic average of F107 (observed).
+        Real F107ObsLast81;    /// Last 81-day arithmetic average of F107 (observed).
+        Real F107AdjCenter81;  /// Centered 81-day arithmetic average of F10.7 (adjusted).
+        Real F107AdjLast81;    /// Last 81-day arithmetic average of F10.7 (adjusted)
+    };
+
+    friend std::ostream& operator<<(std::ostream& anOutputStream, const CSSISpaceWeather& aCSSISpaceWeather);
+
+    bool isDefined() const;
+
+    const Date& accessLastObservationDate() const;
+
+    const Interval& accessObservationInterval() const;
+
+    const Reading& accessObservationAt(const Instant& anInstant) const;
+
+    const Interval& accessDailyPredictionInterval() const;
+
+    const Reading& accessDailyPredictionAt(const Instant& anInstant) const;
+
+    const Interval& accessMonthlyPredictionInterval() const;
+
+    const Reading& accessMonthlyPredictionAt(const Instant& anInstant) const;
+
+    static CSSISpaceWeather Undefined();
+
+    static CSSISpaceWeather Load(const File& aFile);
```

Comment:
Can we add docstrings for these?

Reply:
yeah agreed

Reply:
Me & github copilot are on it

Reply:
Added docstrings to the reference files Bulletin A/B + Finals for good measure

Reply:
Thanks a lot for the alignment effort 🙏